src/activity_handlers.cpp

```diff
@@ -3197,6 +3197,13 @@ void activity_handlers::butcher_do_turn( player_activity * /*act*/, player *p )
 
 void activity_handlers::read_do_turn( player_activity *act, player *p )
 {
+    if( p->fine_detail_vision_mod() > 4 ) {
```

Comment:
Is this magic number used somewhere else? Maybe we should use some constexpr for it?

Reply:
Yes let's make this a constant or an enum, but it's in dozens of places, so let's do that in another PR.